flink-ml-uber/src/main/resources/META-INF/licenses/LICENSE.blas

```diff
@@ -0,0 +1,19 @@
+Copyright 2020, 2021, Ludovic Henry
```

Comment:
Is here as expected ? The `Ludovic Henry` is?

Reply:
This one should be the author of the blas library, thus there should be no problem here.